### ISSUE TITLE
Aborted query notification in Redshift

### ISSUE THREAD

**Issue by andreys70** [awslabs/aws-lambda-redshift-loader#239]
I faced a weird behavior of the Redshift cluster (the support ticket is still open with service team: Case ID 9386087231) where COPY command query sent by the LambdaRedshiftLoader getting aborted first, and then re-written, and competed successfully by the Redshift engine. The problem is that the only Aborted result reported back to the Lambda function and batch status marked as 'error'. I verified in the Redshift that data was loaded successfully in this scenario.
Also, the complexity of this issue is that there is multi cluster scenario, where data loaded to two Redshift clusters and only one is failing with the above scenario. This leads to a question I can't find an answer to in this repo: If I reprocess the 'error' batch does the data loaded to both clusters, or solution is smart enough to load failed batches only for failed Redshift clusters?

**Comment by IanMeyers** [comment]
Yes, if you reprocess the batch it will use the configuration entry that you have set when the reprocess occurs, which if unchanged would load both clusters. You could create a single cluster configuration that you link to the prefix and change the old one if you want to change the load target.

**Comment by andreys70** [comment]
The s3Prefix is a primary key in config table, so it won't let me to create two separate load configurations, with different Redshift clusters, but for the same s3 prefix.
I had to trick the config a bit and created two separate entries with the single Redshift cluster config for each; one prod and one dr. I hope it is going to work:
    1. `bucket/source/schema/table_name/year=*/month=*/day=*/hour=*`
    2. `bucket/source/schema/table_name/year=*/month=*/day=*/*`

**Comment by IanMeyers** [comment]
I think in that case it will potentially use both, or may just select the first one. I would instead change the prefix to a dummy value for the single cluster, and then flip it back to the dual cluster configuration once the reprocess is done.

**Comment by andreys70** [comment]
I used your suggestion, Ian. The approach that I described above did not work, and as you said, it picked up the first matching prefix configuration.
